lane_planner:
  trajectories:
    lane0:
    - - -66.6010153341225
      - -23.234601616808327
    - - -71.39638671757928
      - -21.85221596163142
    - - -75.79371165744983
      - -19.492107875524454
    - - -79.59624291137904
      - -16.259874457621812
    - - -82.63384578420339
      - -12.300133861077098
    - - -84.77061037688874
      - -7.790054717607296
    - - -85.91093253306238
      - -2.931429172758767
    - - -86.00379140648452
      - 2.058355796009111
    - - -85.04503226046322
      - 6.956044812763399
    - - -83.07755236109591
      - 11.542503103263886
    - - -80.18938164701181
      - 15.612521129172626
    - - -76.50974405121956
      - 18.983996172613875
    - - -72.20327570130529
      - 21.50608006418088
    - - -67.46265869006888
      - 23.065928504796503
    - - -62.5
      - 23.59375
    - - -57.5
      - 23.59375
    - - -52.5
      - 23.59375
    - - -47.5
      - 23.59375
    - - -42.5
      - 23.59375
    - - -37.5
      - 23.59375
    - - -32.5
      - 23.59375
    - - -27.5
      - 23.59375
    - - -22.5
      - 23.59375
    - - -17.5
      - 23.59375
    - - -12.5
      - 23.59375
    - - -7.5
      - 23.59375
    - - -2.5
      - 23.59375
    - - 2.5
      - 23.59375
    - - 7.5
      - 23.59375
    - - 12.5
      - 23.59375
    - - 17.5
      - 23.59375
    - - 22.5
      - 23.59375
    - - 27.5
      - 23.59375
    - - 32.5
      - 23.59375
    - - 37.5
      - 23.59375
    - - 42.5
      - 23.59375
    - - 47.5
      - 23.59375
    - - 52.5
      - 23.59375
    - - 57.5
      - 23.59375
    - - 62.5
      - 23.59375
    - - 69.45604481276341
      - 22.54503226046322
    - - 74.04250310326388
      - 20.577552361095904
    - - 78.11252112917262
      - 17.689381647011817
    - - 81.48399617261387
      - 14.009744051219558
    - - 84.00608006418088
      - 9.703275701305293
    - - 85.5659285047965
      - 4.962658690068881
    - - 86.09375
      - 0.0
    - - 85.04503226046322
      - -6.956044812763443
    - - 83.07755236109588
      - -11.542503103263925
    - - 80.1893816470118
      - -15.612521129172645
    - - 76.50974405121954
      - -18.98399617261388
    - - 72.20327570130527
      - -21.50608006418089
    - - 67.46265869006886
      - -23.065928504796506
    - - 62.49999999999999
      - -23.59375
    - - 57.5
      - -23.59375
    - - 52.5
      - -23.59375
    - - 47.5
      - -23.59375
    - - 42.5
      - -23.59375
    - - 37.5
      - -23.59375
    - - 32.5
      - -23.59375
    - - 27.5
      - -23.59375
    - - 22.5
      - -23.59375
    - - 17.5
      - -23.59375
    - - 12.5
      - -23.59375
    - - 7.5
      - -23.59375
    - - 2.5
      - -23.59375
    - - -2.5
      - -23.59375
    - - -7.5
      - -23.59375
    - - -12.5
      - -23.59375
    - - -17.5
      - -23.59375
    - - -22.5
      - -23.59375
    - - -27.5
      - -23.59375
    - - -32.5
      - -23.59375
    - - -37.5
      - -23.59375
    - - -42.5
      - -23.59375
    - - -47.5
      - -23.59375
    - - -52.5
      - -23.59375
    - - -57.5
      - -23.59375
    - - -62.5
      - -23.59375
    lane1:
    - - -69.46984204461282
      - -23.52027908923802
    - - -74.08638598958676
      - -21.62262440738421
    - - -78.22325765350283
      - -18.829800724491612
    - - -81.70919201297423
      - -15.257429953015906
    - - -84.39987261873645
      - -11.053406979100052
    - - -86.18390624559188
      - -6.391776866608675
    - - -86.98743453252555
      - -1.4655294530559755
    - - -86.77719169272231
      - 3.5213903613935007
    - - -85.56188170537244
      - 8.362525860642092
    - - -83.39181797401255
      - 12.857455747667428
    - - -80.35684036941836
      - 16.82009151531616
    - - -76.58259589073732
      - 20.086381444667634
    - - -72.2253369241739
      - 22.52110228816076
    - - -67.46545244947811
      - 24.0234574642051
    - - -62.5
      - 24.53125
    - - -57.5
      - 24.53125
    - - -52.5
      - 24.53125
    - - -47.5
      - 24.53125
    - - -42.5
      - 24.53125
    - - -37.5
      - 24.53125
    - - -32.5
      - 24.53125
    - - -27.5
      - 24.53125
    - - -22.5
      - 24.53125
    - - -17.5
      - 24.53125
    - - -12.5
      - 24.53125
    - - -7.5
      - 24.53125
    - - -2.5
      - 24.53125
    - - 2.5
      - 24.53125
    - - 7.5
      - 24.53125
    - - 12.5
      - 24.53125
    - - 17.5
      - 24.53125
    - - 22.5
      - 24.53125
    - - 27.5
      - 24.53125
    - - 32.5
      - 24.53125
    - - 37.5
      - 24.53125
    - - 42.5
      - 24.53125
    - - 47.5
      - 24.53125
    - - 52.5
      - 24.53125
    - - 57.5
      - 24.53125
    - - 62.5
      - 24.53125
    - - 66.0213903613935
      - 24.277191692722305
    - - 70.86252586064208
      - 23.061881705372446
    - - 75.35745574766742
      - 20.891817974012554
    - - 79.32009151531615
      - 17.85684036941837
    - - 82.58638144466764
      - 14.082595890737329
    - - 85.02110228816076
      - 9.725336924173906
    - - 86.5234574642051
      - 4.965452449478116
    - - 87.03125
      - 0.0
    - - 86.77719169272231
      - -3.5213903613934607
    - - 85.56188170537246
      - -8.362525860642055
    - - 83.39181797401255
      - -12.857455747667412
    - - 80.35684036941838
      - -16.82009151531614
    - - 76.58259589073732
      - -20.086381444667634
    - - 72.2253369241739
      - -22.52110228816076
    - - 67.46545244947811
      - -24.0234574642051
    - - 62.49999999999999
      - -24.53125
    - - 57.5
      - -24.53125
    - - 52.5
      - -24.53125
    - - 47.5
      - -24.53125
    - - 42.5
      - -24.53125
    - - 37.5
      - -24.53125
    - - 32.5
      - -24.53125
    - - 27.5
      - -24.53125
    - - 22.5
      - -24.53125
    - - 17.5
      - -24.53125
    - - 12.5
      - -24.53125
    - - 7.5
      - -24.53125
    - - 2.5
      - -24.53125
    - - -2.5
      - -24.53125
    - - -7.5
      - -24.53125
    - - -12.5
      - -24.53125
    - - -17.5
      - -24.53125
    - - -22.5
      - -24.53125
    - - -27.5
      - -24.53125
    - - -32.5
      - -24.53125
    - - -37.5
      - -24.53125
    - - -42.5
      - -24.53125
    - - -47.5
      - -24.53125
    - - -52.5
      - -24.53125
    - - -57.5
      - -24.53125
    - - -62.5
      - -24.53125
    lane2:
    - - -62.5
      - -25.46875
    - - -57.5
      - -25.46875
    - - -52.5
      - -25.46875
    - - -47.5
      - -25.46875
    - - -42.5
      - -25.46875
    - - -37.5
      - -25.46875
    - - -32.5
      - -25.46875
    - - -27.5
      - -25.46875
    - - -22.5
      - -25.46875
    - - -17.5
      - -25.46875
    - - -12.5
      - -25.46875
    - - -7.5
      - -25.46875
    - - -2.5
      - -25.46875
    - - 2.5
      - -25.46875
    - - 7.5
      - -25.46875
    - - 12.5
      - -25.46875
    - - 17.5
      - -25.46875
    - - 22.5
      - -25.46875
    - - 27.5
      - -25.46875
    - - 32.5
      - -25.46875
    - - 37.5
      - -25.46875
    - - 42.5
      - -25.46875
    - - 47.5
      - -25.46875
    - - 52.5
      - -25.46875
    - - 57.5
      - -25.46875
    - - 62.49999999999999
      - -25.46875
    - - 67.46794420494463
      - -24.979526755706505
    - - 72.24503226018325
      - -23.530651771901425
    - - 76.64774023802732
      - -21.177787247958715
    - - 80.50692696895747
      - -18.011324429289814
    - - 83.67433202772001
      - -14.15291099888537
    - - 86.02827153278645
      - -9.750777673702297
    - - 87.47831294062486
      - -4.974043546523567
    - - 87.96875
      - 0.0
    - - 87.4795267557065
      - 4.967944204944612
    - - 86.03065177190143
      - 9.745032260183244
    - - 83.67778724795872
      - 14.1477402380273
    - - 80.51132442928984
      - 18.006926968957444
    - - 76.65291099888542
      - 21.174332027719984
    - - 72.25077767370234
      - 23.528271532786423
    - - 67.47404354652363
      - 24.978312940624846
    - - 62.5
      - 25.46875
    - - 57.5
      - 25.46875
    - - 52.5
      - 25.46875
    - - 47.5
      - 25.46875
    - - 42.5
      - 25.46875
    - - 37.5
      - 25.46875
    - - 32.5
      - 25.46875
    - - 27.5
      - 25.46875
    - - 22.5
      - 25.46875
    - - 17.5
      - 25.46875
    - - 12.5
      - 25.46875
    - - 7.5
      - 25.46875
    - - 2.5
      - 25.46875
    - - -2.5
      - 25.46875
    - - -7.5
      - 25.46875
    - - -12.5
      - 25.46875
    - - -17.5
      - 25.46875
    - - -22.5
      - 25.46875
    - - -27.5
      - 25.46875
    - - -32.5
      - 25.46875
    - - -37.5
      - 25.46875
    - - -42.5
      - 25.46875
    - - -47.5
      - 25.46875
    - - -52.5
      - 25.46875
    - - -57.5
      - 25.46875
    - - -62.5
      - 25.46875
    - - -67.46794420494462
      - 24.979526755706505
    - - -72.24503226018325
      - 23.530651771901425
    - - -76.64774023802731
      - 21.17778724795872
    - - -80.50692696895746
      - 18.011324429289836
    - - -83.67433202771998
      - 14.152910998885412
    - - -86.02827153278642
      - 9.750777673702334
    - - -87.47831294062485
      - 4.974043546523641
    - - -87.96874924073002
      - 0.006218947995741457
    - - -87.48073908141748
      - -4.961844567158578
    - - -86.03303060803297
      - -9.73928626562969
    - - -83.6812412055004
      - -14.14256863362919
    - - -80.51572081572127
      - -18.00252843498623
    - - -76.65808091589525
      - -21.170875544990164
    - - -72.25652250584446
      - -23.525889890829863
    - - -67.480142591532
      - -24.977097636244867
    lane3:
    - - -62.5
      - -26.40625
    - - -57.5
      - -26.40625
    - - -52.5
      - -26.40625
    - - -47.5
      - -26.40625
    - - -42.5
      - -26.40625
    - - -37.5
      - -26.40625
    - - -32.5
      - -26.40625
    - - -27.5
      - -26.40625
    - - -22.5
      - -26.40625
    - - -17.5
      - -26.40625
    - - -12.5
      - -26.40625
    - - -7.5
      - -26.40625
    - - -2.5
      - -26.40625
    - - 2.5
      - -26.40625
    - - 7.5
      - -26.40625
    - - 12.5
      - -26.40625
    - - 17.5
      - -26.40625
    - - 22.5
      - -26.40625
    - - 27.5
      - -26.40625
    - - 32.5
      - -26.40625
    - - 37.5
      - -26.40625
    - - 42.5
      - -26.40625
    - - 47.5
      - -26.40625
    - - 52.5
      - -26.40625
    - - 57.5
      - -26.40625
    - - 62.49999999999999
      - -26.40625
    - - 67.47017594247941
      - -25.93428985268922
    - - 72.2626875085348
      - -24.535280142545723
    - - 76.70622112924188
      - -22.259230002171716
    - - 80.64193783411378
      - -19.187499333828036
    - - 83.92915112387055
      - -15.429890510720332
    - - 86.45035596326679
      - -11.120723371045221
    - - 88.11542912712642
      - -6.4140338084284965
    - - 88.90625
      - 0.0
    - - 88.43428985268922
      - 4.970175942479409
    - - 87.03528014254572
      - 9.762687508534823
    - - 84.7592300021717
      - 14.206221129241902
    - - 81.68749933382803
      - 18.141937834113783
    - - 77.9298905107203
      - 21.42915112387058
    - - 73.62072337104519
      - 23.950355963266798
    - - 68.91403380842846
      - 25.615429127126422
    - - 62.5
      - 26.40625
    - - 57.5
      - 26.40625
    - - 52.5
      - 26.40625
    - - 47.5
      - 26.40625
    - - 42.5
      - 26.40625
    - - 37.5
      - 26.40625
    - - 32.5
      - 26.40625
    - - 27.5
      - 26.40625
    - - 22.5
      - 26.40625
    - - 17.5
      - 26.40625
    - - 12.5
      - 26.40625
    - - 7.5
      - 26.40625
    - - 2.5
      - 26.40625
    - - -2.5
      - 26.40625
    - - -7.5
      - 26.40625
    - - -12.5
      - 26.40625
    - - -17.5
      - 26.40625
    - - -22.5
      - 26.40625
    - - -27.5
      - 26.40625
    - - -32.5
      - 26.40625
    - - -37.5
      - 26.40625
    - - -42.5
      - 26.40625
    - - -47.5
      - 26.40625
    - - -52.5
      - 26.40625
    - - -57.5
      - 26.40625
    - - -62.5
      - 26.40625
    - - -67.4701759424794
      - 25.93428985268922
    - - -72.26268750853482
      - 24.535280142545716
    - - -76.7062211292419
      - 22.25923000217171
    - - -80.64193783411379
      - 19.18749933382802
    - - -83.92915112387057
      - 15.429890510720309
    - - -86.45035596326679
      - 11.120723371045196
    - - -88.11542912712642
      - 6.414033808428481
    - - -88.86485075362141
      - 1.4780675903925105
    - - -88.6718319468473
      - -3.510733770664272
    - - -87.54327237521436
      - -8.374040112354693
    - - -85.51951363525193
      - -12.938007229050033
    - - -82.67289719710696
      - -17.039491123195376
    - - -79.10517847955225
      - -20.53187976598136
    - - -74.94388949122218
      - -23.29033390470713
    - - -70.33778005944433
      - -25.216249578442007
    - - -65.45150060492982
      - -26.240782824481418
